Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:boron
+
+# Create app directory
+WORKDIR /usr/src/app
+
+# Install app dependencies
+RUN git clone 'https://github.com/mapzen/slack-search-app.git' '/usr/src/app/search-slackbot'
```

Comment:
This should probably switch to using the local code instead of cloning from git. The [Dockerfile](https://github.com/pelias/api/blob/master/Dockerfile#L10-L16) in pelias/api has a good example.